### DISCUSSION TITLE
Is there any way to hide exmaples?

### DISCUSSION THREAD

**Discussion by alkene5** [johnlindquist/kit#1277] [Q&A] [answered]
<img width="768" alt="image" src="https://github.com/johnlindquist/kit/assets/61105068/e93c74b7-218f-48eb-9883-78e9930c492d">

**Comment by johnlindquist** [comment, marked as answer]
Do you see this script?

![CleanShot 2023-06-13 at 10 44 43](https://github.com/johnlindquist/kit/assets/36073/0bae5f11-3b89-4ac3-8c92-6fa3e297a273)

If not, add this to your `~/.kenv/.env` (that's what the script does):

```bash
KIT_EXCLUDE_KENVS=examples
```
